packages/cacao/src/siwx/siwe.ts

```diff
@@ -13,22 +14,31 @@ export class SiweMessage extends SiwxMessage {
     return super.toMessage('Ethereum')
   }
 
+  toMessageEip55(): string {
+    const tmpAddress = this.address
+    this.address = checksumAddress(this.address as `0x${string}`)
+    const msg = super.toMessage('Ethereum')
+    this.address = tmpAddress
+    return msg
+  }
+
   /**
    * This method parses all the fields in the object and creates a sign
    * message according with the type defined.
    * @returns {string} Returns a message ready to be signed according with the
    * type defined in the object.
    */
-  signMessage(): string {
+  signMessage(eip55?: boolean): string {
```

Comment:
Could we use an options object here to be more explicit and in case we need to add more options in the future?
```suggestion
  signMessage(options: {eip55?: boolean} = {}): string {
```